components/package.json

```diff
@@ -49,43 +46,26 @@
   "dependencies": {
     "@types/leaflet": "^1.9.12",
     "@types/leaflet-draw": "^1.0.11",
-    "autocompleter": "^7.0.1",
     "css-loader": "^7.1.2",
     "formiojs": "^4.14.6",
     "leaflet": "^1.9.4",
     "leaflet-draw": "^1.0.4",
     "leaflet-geosearch": "^4.0.0",
     "lodash": "^4.17.21",
-    "native-promise-only": "^0.8.1",
-    "path-browserify": "^1.0.1",
     "webpack": "^5.75.0"
```

Comment:
Is `webpack` a runtime dependency, or should it be in the devDependencies? Likewise with `css-loader`?

Reply:
As far as I know, `webpack` is a bundler so it should be in devDependencies. I believe `css-loader` should also be a devDependency. It's building fine with this package.json but we don't have tests in place to know if anything will truly be broken. The cypress kitchen sink is passing but I'm not sure if that includes all of the new formio components.

Reply:
I'll move them into the `devDependencies`.